addons/core/XEH_PREP.sqf

```diff
@@ -177,6 +179,8 @@ PREP_SUB(hint,showRadioVolume);
 PREP(setChannelFrequency);
 PREP(setRadioOwner);
 PREP(setLrSpeakers);
+PREP(setSwHalfDuplexOverride);
+PREP(setLrHalfDuplexOverride);
```

Comment:
You can probably put both into one function. LR Radio is always an Array and SR Radio is always a string. That way you can detect inside the function what was passed.
I think having it in one function is easier to use.